submissions/lawalLolade/HTML/productPage.html

```diff
@@ -77,7 +83,7 @@ <h3>Description</h3>
                     to white, the colors of carnations are truly spectacular.
                 </p> 
             </div>
-        </div>
+        </div> -->
```

Comment:
Is there a reason why you comment from line 62 - 86?

Reply:
to avoid my Js being cumbersome @adaugochi for both sides